src/metrics/prometheus/cluster_process_role.rs

```diff
@@ -116,7 +117,13 @@ impl StaticMetric<GaugeVec> for ClusterProcessRoleFreq {
                 "hz" => self.hz,
                 "roughness" => self.roughness,
                 // Impossible case
-                &_ => -1.0,
+                &_ => {
+                    warn!(
+                        "ClusterProcessRoleFreq::set() went through irregular case for {}",
+                        name
+                    );
+                    -1.0
```

Comment:
returning an option would be a better ap than returning a magic number

Reply:
It makes sense, made the changes